src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -148,28 +146,24 @@ public override Collection<Attribute> GetAttributes()
                 }
 
                 string connectionString = Context.GetMetadataValue<string>("connection");
-                if (!string.IsNullOrEmpty(connectionString))
-                {
-                    connectionString = _nameResolver.Resolve(connectionString);
-                }
 
                 if (Context.IsTrigger)
                 {
                     var attribute = new EventHubTriggerAttribute(eventHubName);
+                    attribute.Connection = connectionString;
```

Comment:
nit: is there some scenario where we wouldn't set a connection string on create? Why isn't this set in the constructor?

Reply:
There is no constructor that takes the connection. Strictly speaking this is because its not required in webjobs scenarios see https://github.com/Azure/azure-webjobs-sdk/blob/9700db45f51717911c11bb7c0973244de02f2183/test/Microsoft.Azure.WebJobs.Host.EndToEndTests/EventHubEndToEndTests.cs#L133

Reply:
I see, we let folks register Event Hubs in the Configuration Provider which allows for folks to not have to specify the connection string in their Functions: https://github.com/Azure/azure-webjobs-sdk/blob/9700db45f51717911c11bb7c0973244de02f2183/src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubConfiguration.cs#L121